src/components/Calculator.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable no-useless-constructor */
+/* eslint-disable react/prefer-stateless-function */
+import React, { Component } from 'react';
+import './calculator.css';
+
+class CalcComponets extends Component {
```

Comment:
## Optional
- [ ] I suggest that you call this `calculator` it can make more sense